packages/components/src/input-control/types.ts

```diff
@@ -27,9 +27,24 @@ export type Size = 'default' | 'small' | '__unstable-large';
 
 interface BaseProps {
 	__unstableInputWidth?: CSSProperties[ 'width' ];
+	/**
+	 * If true, the label will only be visible to screen readers.
+	 *
+	 * @default false
+	 */
 	hideLabelFromVision?: boolean;
 	isFocused: boolean;
```

Comment:
Should we also add docs for `isFocused` (including the `false` default value) ?

Reply:
Sure thing. Does this description look correct? https://github.com/WordPress/gutenberg/pull/40119/commits/128618aee40f84d2cbebb041fe5502a2ded7a7e9

Reply:
Yep, looks good!